packages/browser/src/posthog-core.ts

```diff
@@ -163,7 +163,7 @@ export const defaultConfig = (defaults?: ConfigDefaults): PostHogConfig => ({
     disable_external_dependency_loading: false,
     enable_recording_console_log: undefined, // When undefined, it falls back to the server-side setting
     secure_cookie: window?.location?.protocol === 'https:',
-    ip: true,
+    ip: false,
```

Comment:
As mentioned, I don't think it's a breaking change because it's not doing anything on the servers right now.